### PR TITLE
Add CONTRIBUTING.md to the project

### LINKED ISSUE #1: Plan the CONTRIBUTING.md file
This project is going to be great :rocket: but before we open it up to the public we need to setup some guidelines for external contributions. 

@brooksseahorn What do you think about this :bulb: idea?

### PR DESCRIPTION
**Pull request by mattdavis0351**
This PR creates and adds a `.github/CONTRIBUTING.md` file to the repo.

### CODE REVIEW

**Comment by mattdavis0351** [comment]
@brooksseahorn I added what I think is the proper line, can you take a look?

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -1 +1,4 @@
 # Guide for Contributions
+
+
+To contibueute please visit this [link](https://nowhere.xyz)
```
Comment (brooksseahorn) [inline .github/CONTRIBUTING.md]: Bad grammar yo!

Comment (mattdavis0351) [reply .github/CONTRIBUTING.md]:
```suggestion
To contribute please visit this [link](https://nowhere.xyz)
```

Comment (mattdavis0351) [reply .github/CONTRIBUTING.md]: @brooksseahorn hey, is the above better?  if it is, go ahead and commit those changes

Comment (brooksseahorn) [reply .github/CONTRIBUTING.md]: Looking good!

**Comment by mattdavis0351** [comment]
hey @brooksseahorn , turns out our manager.... (he who has no name...) decided that **you** specifically should add out some real step to contribute.

Can you make that happen by friday feb 900

File: .github/CONTRIBUTING.md [hunk 2]
```diff
@@ -2,3 +2,5 @@
 
 
 To contribute please visit this [link](https://nowhere.xyz)
+
+I'm helpful.
```
Comment (mattdavis0351) [inline .github/CONTRIBUTING.md:6]: yes you are

**Review by brooksseahorn — APPROVED** [review]

**Pull request merged**